Dockerfile

```diff
@@ -45,7 +46,9 @@ RUN curl -O https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8/hdf5-1.8.19/
     && /bin/sh configure --prefix=/usr/local \
     && make \
```

Comment:
More of a comment/suggestion than something that needs to be done -- GNU Make has a `-j` argument that will allow parts of a Makefile which have all dependencies met to be run on parallel processors: https://www.gnu.org/software/make/manual/html_node/Parallel.html
This can help to speed up build times.